docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.1'
+services:
+  sidekick:
+    build:
+      context: .
+    ports: 
+      - 8082:8080
+    environment:
+      DATABASE_URL: postgres://sidekick:sidekick@db:5432/sidekick
+      LOG_LEVEL: debug
+      NODE_ENV: development
+      PORT: 8080
+      SESSION_SECRET: b3c542a1-79ce-46a1-b958-06e352eba249
```

Comment:
Can this be read from the environment variables, as it should be different per installation. Think you can read in variables from the host shell using
```
      SESSION_SECRET: ${SESSION_SECRET}
```

Reply:
Yeah I guess that would be the best.